runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/grpcclient/GrpcCommitWorkStream.java

```diff
@@ -56,7 +56,8 @@ final class GrpcCommitWorkStream
   private final int streamingRpcBatchLimit;
 
   private GrpcCommitWorkStream(
-      CloudWindmillServiceV1Alpha1Grpc.CloudWindmillServiceV1Alpha1Stub stub,
+      Function<StreamObserver<StreamingCommitResponse>, StreamObserver<StreamingCommitWorkRequest>>
+          commitWorkRpc,
```

Comment:
nit: commitWorkRpc is a confusing name to me for a function.  What about createCommitWorkRpcFn or startCommitWorkRpcFn etc?

ditto for the other files

Reply:
done.